<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookManager</class>
 <widget class="QMainWindow" name="bookManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BookManager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/layers.png</normaloff>images/layers.png</iconset>
  </property>
  <widget class="QTreeWidget" name="bookTree">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="allColumnsShowFocus">
    <bool>false</bool>
   </property>
   <attribute name="headerCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>199</number>
   </attribute>
   <attribute name="headerMinimumSectionSize">
    <number>0</number>
   </attribute>
   <attribute name="headerShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>进度</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>时间</string>
    </property>
   </column>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionSelectDir"/>
   <addaction name="actionOpenFile"/>
   <addaction name="actionSyncTree"/>
   <addaction name="separator"/>
   <addaction name="actionEdit"/>
   <addaction name="actionFind"/>
  </widget>
  <action name="actionSelectDir">
   <property name="icon">
    <iconset>
     <normaloff>images/folder.png</normaloff>images/folder.png</iconset>
   </property>
   <property name="text">
    <string>选择所在文件夹</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset>
     <normaloff>images/book_1.png</normaloff>images/book_1.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionSyncTree">
   <property name="icon">
    <iconset>
     <normaloff>images/cloud_upload.png</normaloff>images/cloud_upload.png</iconset>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset>
     <normaloff>images/pen_3.png</normaloff>images/pen_3.png</iconset>
   </property>
   <property name="text">
    <string>编辑信息</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset>
     <normaloff>images/magnifying_glass.png</normaloff>images/magnifying_glass.png</iconset>
   </property>
   <property name="text">
    <string>查找文件</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
